hera_cal/lst_stack/averaging.py

```diff
@@ -355,6 +355,32 @@ def reduce_lst_bins(
     return o
 
 
+def _get_convolved_flags(
+    raw_flags: np.ndarray,
+    max_allowed_gap_size: float | int,
+    max_convolved_flag_frac: float,
+    axis: int = -1,
+):
+    # Now we have all the per-night models. Next thing to do is to make sure we flag
+    # anything where the flag gaps are too big.
+    convolution_kernel = np.append(
```

Comment:
I'd clarify that this convolution is triangular with base width = 2* max_allowed_gap_size - 1.

Reply:
I added a docstring mentioning this.